### PR TITLE
Added travis and coveralls configurations.

### PR DESCRIPTION
**Pull request by ValterSantosMatos**
Hi there, as discussed  #22  the PR follows
- Added travis and coveralls configurations only when using npm tasks (couldn't find the coverage task when using gulp).
- Fixed a very small typo in the Readme file.

If you like I could also change:
- Update the following libraries to the latest version
  - @types/node => from 6.0.0 to 7.0.0
  - ts-node => from 2.0.0 to 3.0.0
  - tslint => from 4.0.0 to 5.0.0
- Add the travis and the coveralls links to the readme file
- Add the badges for travis and coveralls to the default Readme file

Let me know if its all good,
Thanks,
Valter

### CODE REVIEW

**Comment by ValterSantosMatos** [comment]
#22

**Comment by ospatil** [comment]
Looks great 👍 . Regarding other changes you proposed, please add those to this pull request.

File: generators/app/templates/src/greeter.ts [hunk 1]
```diff
@@ -8,4 +8,4 @@ export class Greeter {
   public greet() {
     return "Bonjour, " + this.greeting + "!";
   }
-};
+}
```
Comment (ValterSantosMatos) [inline generators/app/templates/src/greeter.ts]: This semi colon will give a linting error.

**Comment by ValterSantosMatos** [comment]
Here you go :)

**Pull request merged**